charts/soroban-rpc/pubnet-values.yaml

```diff
@@ -0,0 +1,224 @@
+        name: "bootes"
+        publickey: "GCVJ4Z6TI6Z2SOGENSPXDQ2U4RKH3CNQKYUHNSSPYFPNWTLGS6EBH7I2"
+      - address: "hercules.publicnode.org"
+        history: "curl -sf https://hercules-history.publicnode.org/{0} -o {1}"
+        homeDomain: "publicnode.org"
+        name: "hercules"
+        publickey: "GBLJNN3AVZZPG2FYAYTYQKECNWTQYYUUY2KVFN2OUKZKBULXIXBZ4FCT"
+      - address: "lyra.publicnode.org"
+        history: "curl -sf https://lyra-history.publicnode.org/{0} -o {1}"
+        homeDomain: "publicnode.org"
+        name: "lyra"
+        publickey: "GCIXVKNFPKWVMKJKVK2V4NK7D4TC6W3BUMXSIJ365QUAXWBRPPJXIR2Z" 
+      - address: "v1.stellar.lobstr.co"
+        history: "curl -sf https://stellar-archive-1-lobstr.s3.amazonaws.com/{0} -o {1}"
+        homeDomain: "lobstr.co"
+        name: "lobstr_1_eu"
+        publickey: "GCFONE23AB7Y6C5YZOMKUKGETPIAJA4QOYLS5VNS4JHBGKRZCPYHDLW7"
+      - address: "v2.stellar.lobstr.co"
+        history: "curl -sf https://stellar-archive-2-lobstr.s3.amazonaws.com/{0} -o {1}"
+        homeDomain: "lobstr.co"
+        name: "lobstr_2_eu"
+        publickey: "GCB2VSADESRV2DDTIVTFLBDI562K6KE3KMKILBHUHUWFXCUBHGQDI7VL"
+      - address: "v3.stellar.lobstr.co"
+        history: "curl -sf https://stellar-archive-3-lobstr.s3.amazonaws.com/{0} -o {1}"
+        homeDomain: "lobstr.co"
+        name: "lobstr_3_north_america"
+        publickey: "GD5QWEVV4GZZTQP46BRXV5CUMMMLP4JTGFD7FWYJJWRL54CELY6JGQ63"
+      - address: "v4.stellar.lobstr.co"
+        history: "curl -sf https://stellar-archive-4-lobstr.s3.amazonaws.com/{0} -o {1}"
+        homeDomain: "lobstr.co"
+        name: "lobstr_4_asia"
+        publickey: "GA7TEPCBDQKI7JQLQ34ZURRMK44DVYCIGVXQQWNSWAEQR6KB4FMCBT7J"
+      - address: "v5.stellar.lobstr.co"
+        history: "curl -sf https://stellar-archive-5-lobstr.s3.amazonaws.com/{0} -o {1}"
+        homeDomain: "lobstr.co"
+        name: "lobstr_5_au"
+        publickey: "GA5STBMV6QDXFDGD62MEHLLHZTPDI77U3PFOD2SELU5RJDHQWBR5NNK7"
+      - address: "stellar1.franklintempleton.com:11625"
+        history: "curl -sf https://stellar-history-usw.franklintempleton.com/azuswshf401/{0} -o {1}"
+        homeDomain: "www.franklintempleton.com"
+        name: "FT_SCV_1"
+        publickey: "GARYGQ5F2IJEBCZJCBNPWNWVDOFK7IBOHLJKKSG2TMHDQKEEC6P4PE4V"
+      - address: "stellar2.franklintempleton.com:11625"
+        history: "curl -sf https://stellar-history-usc.franklintempleton.com/azuscshf401/{0} -o {1}"
+        homeDomain: "www.franklintempleton.com"
+        name: "FT_SCV_2"
+        publickey: "GCMSM2VFZGRPTZKPH5OABHGH4F3AVS6XTNJXDGCZ3MKCOSUBH3FL6DOB"
+      - address: "stellar3.franklintempleton.com:11625"
+        history: "curl -sf https://stellar-history-ins.franklintempleton.com/azinsshf401/{0} -o {1}"
+        homeDomain: "www.franklintempleton.com"
+        name: "FT_SCV_3"
+        publickey: "GA7DV63PBUUWNUFAF4GAZVXU2OZMYRATDLKTC7VTCG7AU4XUPN5VRX4A"
+  ingress:
+    host: yourhost.com
```

Comment:
Maybe `example.com` is better for consistency with testnet values?

Reply:
yes, https://github.com/stellar/helm-charts/pull/78/commits/fe36d488d7035ac3d9b45c28c23e85fc046e5c31